x-pack/plugins/reporting/server/routes/lib/job_response_handler.ts

```diff
@@ -46,7 +45,7 @@ export function downloadJobResponseHandlerFactory(reporting: ReportingCore) {
         });
       }
 
-      const payload = getDocumentPayload(doc);
+      const payload = await getDocumentPayload(doc);
```

Comment:
I'm not sure we don't just pass `getDocumentPayload` the `user` and `docId`. It seems like there is an opportunity here to remove an unnecessary roundtrip to ES.

Reply:
Based on this: https://github.com/elastic/kibana/pull/106821/commits/5ebf2ce417d34701989b4b28c5c117b51617df1b#r682644195 we need to keep the `.get` above. If we did the optimization, it would mean adding more logic to the content stream which we don't want to do.